src/adaptive_graph_of_thoughts/app_setup.py

```
@@ -108,20 +154,176 @@
+        return RedirectResponse("/dashboard", status_code=303)
+
+    @app.get("/dashboard", response_class=HTMLResponse)
+    async def dashboard(request: Request, _=Depends(get_basic_auth)):
+        return templates.TemplateResponse(
+            "dashboard.html",
+            {"request": request, "config_yaml": yaml_path.read_text()},
+        )
+
+    @app.post("/dashboard/save_config")
+    async def dashboard_save_config(
+        payload: dict = Body(...), _=Depends(get_basic_auth)
```

Ruff (0.11.9)

Do not perform function call `Body` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)